lib/index.js

```diff
@@ -31,15 +31,15 @@ internals.implementation = function (server, options) {
       if (!token && request.auth.mode === 'optional') {
         return reply.continue({ credentials: {} });
       }
-      
+
       if (!token) {
-        return reply(Boom.unauthorized('Missing auth token'));
+        return reply(Boom.unauthorized(null, 'Token'));
```

Comment:
This is what enables additional strategies to be attempted.
From hapi documentation http://hapijs.com/api#serverauthschemename-scheme:

`If the err returned by the reply() method includes a message, no additional strategies will be attempted. If the err does not include a message but does include a scheme name (e.g. Boom.unauthorized(null, 'Custom')), additional strategies will be attempted in order of preference.`